index.html

```diff
@@ -11329,8 +11329,8 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 			<div class="state-description">
 				<p><a>Indicates</a> that the <a>element</a> is <a>perceivable</a> but disabled, so it is not editable or otherwise <a>operable</a>. See related <sref>aria-hidden</sref> and <pref>aria-readonly</pref>.</p>
 				<p>For example, irrelevant options in a radio group may be disabled. Disabled elements might not receive focus from the tab order. For some disabled elements, applications might choose not to support navigation to descendants. In addition to setting the <sref>aria-disabled</sref> attribute, authors SHOULD change the appearance (grayed out, etc.) to indicate that the item has been disabled.</p>
-				<p>The <a>state</a> of being disabled applies to the element with aria-disabled and all focusable descendant elements of the element on which the <sref>aria-disabled</sref> <a>attribute</a> is applied.</p>
-                <p class="note">While <sref>aria-disabled</sref> and proper scripting can successfully disable an element with role <rref>link</rref>, fully disabling a host language equivalent can be problematic. Authors are advised not to use <sref>aria-disabled</sref> on elements that cannot be disabled through features of the host language alone.</p>
+				<p>The <a>state</a> of being disabled applies to the element which the <sref>aria-disabled</sref> <a>attribute</a> is specified, and any descendants of that element which inherit from the <rref>input</rref> abstract role, to which the <sref>aria-disabled</sref> <a>attribute</a> can be applied. For instance, it would be expected for <rref>textbox</rref>, <rref>combobox</rref>, or their host language equivalents to be exposed in the disabled state if they are descendants of an ancestor element with a specified <sref>aria-disabled</sref> attribute set to the "true" value. However, User Agents MUST NOT set a <rref>link</rref>, or its host language equivalent, to the disabled state if it is a descendant of an ancestor element with a specified <sref>aria-disabled</sref> attribute set to the "true" value</p>
```

Comment:
Mostly editorial but IMO, normative statements should not follow inline behind a non-normative "for instance" example. I've added a line break, among another minor punctuation change (closing period) and a grammatical change ('on which').

```suggestion
				<p>The <a>state</a> of being disabled applies to the element on which the <sref>aria-disabled</sref> <a>attribute</a> is specified, and any descendants of that element which inherit from the <rref>input</rref> abstract role, to which the <sref>aria-disabled</sref> <a>attribute</a> can be applied. For instance, it would be expected for <rref>textbox</rref>, <rref>combobox</rref>, or their host language equivalents to be exposed in the disabled state if they are descendants of an ancestor element with a specified <sref>aria-disabled</sref> attribute set to the "true" value.</p>
				<p>User Agents MUST NOT set a <rref>link</rref>, or its host language equivalent, to the disabled state if it is a descendant of an ancestor element with a specified <sref>aria-disabled</sref> attribute set to the "true" value.</p>
```

Reply:
thank you